src/Workspaces/Core/Portable/FindSymbols/SyntaxTree/SyntaxTreeIndex.ContextInfo.cs

```diff
@@ -24,7 +24,8 @@ private struct ContextInfo
                 bool containsThisConstructorInitializer,
                 bool containsBaseConstructorInitializer,
                 bool containsElementAccessExpression,
-                bool containsIndexerMemberCref) :
+                bool containsIndexerMemberCref,
+                bool containsDeconstruction) :
```

Comment:
need to update the 'version' of SyntaxTreeINdex.  Otherwise, existing indices will not be recreated and it will look like no files contain deconstructions.

Reply:
Updated `SerializationFormat` number.

---
In reply to: [148396504](https://github.com/dotnet/roslyn/pull/22934#discussion_r148396504) [](ancestors = 148396504)